zilencer/views.py

```diff
@@ -114,22 +116,47 @@ def deactivate_remote_server(
     return json_success(request)
 
 
+# The Pydantic.StringConstraints does not have validation for the string to be
+# of the specified length. So, we need to create a custom validator for that.
+
+
+def check_string_fixed_length(string: str, length: int) -> Optional[str]:
+    if len(string) != length:
+        raise PydanticCustomError(
+            "string_fixed_length",
+            "",
+            {
+                "length": length,
+            },
+        )
+    return string
```

Comment:
It might be better for this to live in `typed_endpoint`, since I'm sure other views files will want it.

Reply:
I guess a new `typed_endpoint_validators.py` might be better, going with that.